### ISSUE TITLE
Load plugin settings from file

### ISSUE THREAD

**Issue by ignazioc** [benoitsan/BBUncrustifyPlugin-Xcode#107]
I think it will be nice if the settings could be loaded from a configuration file inside the project.
This because in some project you want to format on save, and some other project you don't.

**Comment by benoitsan** [comment]
Yeah, it would be a nice feature. I will work on it in a future update.
